wpk/run.sh

```diff
@@ -2,36 +2,141 @@
+    echo "    -wr,  --wpk-key-region       [Optional] AWS Region where secrets are stored."
+    echo "    -h,   --help                 Show this help."
+    echo
+    exit ${1}
+}
 
 main() {
 
+    while [ -n "${1}" ]
+    do
+        case "${1}" in
+        "-b"|"--branch")
+            if [ -n "${2}" ]; then
+                REFERENCE="$(echo ${2} | cut -d'/' -f2)"
+                shift 2
+            else
+                echo "ERROR: Missing branch."
+                help 1
+            fi
+            ;;
+        "-o"|"--output")
+            if [ -n "${2}" ]; then
+                OUT_NAME="${2}"
+                shift 2
+            else
+                echo "ERROR: Missing output name."
+                help 1
+            fi
+            ;;
+        "-r"|"--revision")
+            if [ -n "${2}" ]; then
+                REVISION="${2}"
+                shift 2
+            fi
+            ;;
+        "-p"|"--path")
+            if [ -n "${2}" ]; then
+                INSTALLATION_PATH="${2}"
+                shift 2
+            fi
+            ;;
+        "-pn"|"--package-name")
+            if [ -n "${2}" ]; then
+                PKG_NAME="${2}"
+                HAVE_PKG_NAME=true
+                shift 2
+            fi
+            ;;
+        "-j"|"--jobs")
+            if [ -n "${2}" ]; then
+                JOBS="${2}"
+                shift 2
+            fi
+            ;;
+        "-c"|"--checksum")
+            CHECKSUM="yes"
+            shift 1
+            ;;
+        "-wk"|"--wpk-key")
+            if [ -n "${2}" ]; then
+                AWS_WPK_KEY="${2}"
+                shift 2
+            fi
+            ;;
+        "-wc"|"--wpk-cert")
+            if [ -n "${2}" ]; then
+                AWS_WPK_CERT="${2}"
+                shift 2
+            fi
+            ;;
+      "-wr"|"--wpk-key-region")
+            if [ -n "${2}" ]; then
+                AWS_REGION="${2}"
+                shift 2
+            fi
+          ;;
+        "-h"|"--help")
+            help 0
+            ;;
+        *)
+            help 1
+        esac
+    done
+
+    if [ -n "${AWS_WPK_CERT}" ] && [ -n "${AWS_WPK_KEY}" ]; then
+        mkdir -p ${KEYPATH}
+        aws --region=${AWS_REGION} secretsmanager get-secret-value --secret-id ${AWS_WPK_CERT} | jq . > wpkcert.pem.json
+        jq .SecretString wpkcert.pem.json | tr -d '"' | sed 's|\\n|\n|g' > ${WPKCERT}
+        rm -f wpkcert.pem.json
+        aws --region=${AWS_REGION} secretsmanager get-secret-value --secret-id ${AWS_WPK_KEY} | jq . > wpkcert.key.json
+        jq .SecretString wpkcert.key.json | tr -d '"' | sed 's|\\n|\n|g' > ${WPKKEY}
+        rm -f wpkcert.key.json
+    fi
+
+
     NO_COMPILE=false
     # Get Wazuh
-    git clone $REPOSITORY $DIRECTORY || exit 1
-    cd $DIRECTORY
-    git checkout $BRANCH
+    git clone ${REPOSITORY} ${DIRECTORY} || exit 1
+    cd ${DIRECTORY}
+    git checkout ${REFERENCE}
```

Comment:
```suggestion
    git clone ${REPOSITORY} ${DIRECTORY} -b ${REFERENCE} --depth=1 || exit 1
    cd ${DIRECTORY}
```

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/467/commits/1c82879d25119ef9aeb3808e1a4f12747d8edbf0